pages/k8s/how-to-clouds.md

```diff
@@ -0,0 +1,35 @@
+---
+wrapper_template: "templates/docs/markdown.html"
+markdown_includes:
+  nav: "kubernetes/docs/shared/_side-navigation.md"
+context:
+  title: "Integrate Charmed Kubernetes with your cloud"
+  description: How to use cloud integrators
+keywords: Cloud, cluster, storage
+tags: [operating]
+sidebar: k8smain-sidebar
+permalink: how-to-clouds.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+Charmed Kubernetes is designed to be cloud-agnostic. It can be deployed on any cloud or substrate supported by Juju. However, there are specific features of clouds (e.g. storage and load-balancers) which can more effectively be integrated for Charmed Kubernetes. This integration is provided by a series of substrate specific charms. Their use is described in the pages linked below
```

Comment:
Missing punctuation at the end of this line. I'm on the fence between `:` or `.` so I just picked one for my suggestion.

```suggestion
Charmed Kubernetes is designed to be cloud-agnostic. It can be deployed on any cloud or substrate supported by Juju. However, there are specific features of clouds (e.g. storage and load-balancers) which can more effectively be integrated for Charmed Kubernetes. This integration is provided by a series of substrate specific charms. Their use is described in the pages linked below:
```